miq_version/__init__.py

```diff
@@ -31,8 +34,8 @@ def parse(self, vstring):
         if vstring == 'darga-5':
             vstring = '5.6.3'
 
-        components = filter(lambda x: x and x != '.',
-                            self.component_re.findall(vstring))
+        components = list(filter(lambda x: x and x != '.',
+                            self.component_re.findall(vstring)))
```

Comment:
I would say maybe replace this with a list comprehension?

Reply:
I will do that in the next PR